tests/NHSD.BuyingCatalouge.Ordering.Api.Testing.Data/Entities/OrderEntity.cs

```diff
@@ -1,99 +1,130 @@
-        }
-
-        public static async Task<string> FetchLastUpdatedByNameFromOrderId(string connectionString, string orderId)
-        {
+        }
+
+        public static async Task<string> FetchLastUpdatedByNameFromOrderId(string connectionString, string orderId)
+        {
             return (await SqlRunner.QueryFirstAsync<string>(connectionString, @"SELECT
                          LastUpdatedByName
                          FROM dbo.[Order]
                          WHERE OrderId = @orderId;", new { orderId }));
-        }
-
+        }
+
         public static async Task<OrderEntity> FetchOrderByOrderId(string connectionString, string orderId)
         {
             return (await SqlRunner.QueryFirstAsync<OrderEntity>(connectionString, @"SELECT
                           OrderId,
-                          [Description],
+                          [Description],
                           OrganisationId,
+                          OrganisationName,
+                          OrganisationOdsCode,
                           OrderStatusId,
                           Created,
                           LastUpdated,
                           LastUpdatedBy,
                           LastUpdatedByName
-                         FROM dbo.[Order]
-                         WHERE OrderId = @orderId;", new { orderId }));
-        }
-    }
-}
+                         FROM [Order]
+                         WHERE [OrderId] = @orderId", new { orderId }));
+        }
+
+        public static async Task<ContactEntity> FetchPrimaryContactByOrderId(string connectionString, string orderId)
+        {
+            return (await SqlRunner.QueryFirstAsync<ContactEntity>(connectionString, $@"SELECT
+                        [ContactId],
+                        [FirstName],
+                        [LastName],
+                        [Email],
+                        [Phone] 
+                        FROM [CatalogueOrdering].[dbo].[Order] od join [CatalogueOrdering].[dbo].[Contact] ct on od.organisationContactId = ct.ContactId
+                        WHERE [OrderId] = @orderId", new { orderId }));
+        }
+
+        public static async Task<AddressEntity> FetchOrganisationAddressByOrderId(string connectionString, string orderId)
+        {
+            return (await SqlRunner.QueryFirstAsync<AddressEntity>(connectionString, $@"SELECT  
+                        [AddressId],
+                        [Line1],
+                        [Line2],
+                        [Line3],
+                        [Line4],
+                        [Line5],
+                        [Town],
+                        [County],
+                        [Postcode], 
+                        [Country]
+                        FROM [CatalogueOrdering].[dbo].[Order] od join [CatalogueOrdering].[dbo].[Address] ad on od.organisationAddressId = ad.AddressId
+                        WHERE [OrderId] = @orderId", new { orderId }));
```

Comment:
As above on the SQL formatting here, for example:

```sql
SELECT ad.AddressId, ad.Line1, ad.Line2, ad.Line3, ad.Line4, ad.Line5,
       ad.Town, ad.County, ad.Postcode, ad.Country
  FROM dbo.[Order] AS od
       INNER JOIN dbo.[Address] AS ad
       ON ad.AddressId = od.organisationAddressId
 WHERE od.OrderId = @orderId;
```
I'd also be inclined just to use single letter table aliases here too as it would be fairly clear that `o` corresponds to [Order] and `a` corresponds to `[Address]`.

Reply:
See comment above, no longer using a join to access this information